purefa/metadata.csv

```diff
@@ -1,37 +1,64 @@
 metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
```

Comment:
Minor nit that metric descriptions should all end with periods, but this is not required formatting.

Reply:
fixed!